streaming-binlog/src/main/java/org/openmrs/analytics/model/GeneralConfiguration.java

```diff
@@ -24,6 +31,17 @@
 	@Setter
 	private LinkedHashMap<String, EventConfiguration> eventConfigurations;
 	
-	// TODO add Debezium config to the model + json
+	@Getter
+	@Setter
```

Comment:
if all fields need these, you can clean it up and move these to the class level.

Reply:
here we are using lombok which automatically creates the setter and getters for the fields at the class level so that we dont need to hard code them as its done with the` EventConfigurations `above

Reply:
I think what @kaweesi is saying is that if all attributes have these two `@Getter/@Setter` annotation, we can remove all those and only have one at the class level. I think this is a good suggestion, also for the `EventConfiguration` class above.

Reply:
oh i see,, its clear , 
sounds good